app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,9 @@
+.flash_success {
+  background-color: green;
```

Comment:
@j-sm-n: For the color, just something that @JeanJoeris and I went over, maybe add a variable to our custom scss like this:
$green-color: #green; (at the very top of our css file)
and add $green-color; to background-color, like this :
background-color: $green-color;